rn/Teacher/src/modules/inbox/__tests__/__snapshots__/Compose.test.js.snap

```diff
@@ -284,15 +284,96 @@ exports[`Compose renders after picking a course 1`] = `
                   "fontSize": 16,
                   "lineHeight": 19,
                 },
-                undefined,
+                Object {
+                  "color": "#2D3B45",
+                },
               ]
             }
           >
-            Select a Course
+            Learn React Native
```

Comment:
The changes in this file seem odd. Do you know what they are about?

Reply:
These changes happened after I updated enzyme to work with `React.Fragment`. I think these changes are actually correct and that what was there before was not. The test is testing something after choosing a course, but the original snapshot still said `Select a Course`. So it seemed like the enzyme update actually made the test work the way it was intended.